src/main/java/org/jenkinsci/plugins/displayurlapi/DefaultDisplayURLProviderGlobalConfiguration.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2023 CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.jenkinsci.plugins.displayurlapi;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import hudson.Extension;
+import hudson.ExtensionList;
+import hudson.Util;
+import hudson.util.ListBoxModel;
+import java.util.Objects;
+import jenkins.model.GlobalConfiguration;
+import org.jenkinsci.Symbol;
+import org.jenkinsci.plugins.displayurlapi.user.PreferredProviderUserProperty;
+import org.kohsuke.stapler.DataBoundSetter;
+
+@Extension
+@Symbol("defaultNotificationUrl")
```

Comment:
I forgot to add this previously. I just made the name match what shows up in the UI.

Reply:
(let me know if you think it should be changed to `defaultDisplayUrlProvider` through or similar, or if you think we should support both `Url` and `URL`, etc.)

Reply:
I personally prefer defaultDisplayUrlProvider, but that's just a personal preference.

Reply:
Fair enough, probably best to mirror the class name instead of the UI text anyway.
```suggestion
@Symbol("defaultDisplayUrlProvider")
```